src/components/demo/shapesdemos.tsx

```diff
@@ -21,6 +22,7 @@ export const ShapesDemos = () => {
       <LiveEdit code={circleExample} scope={scope} />
       <LiveEdit code={donutExample} scope={scope} />
       <LiveEdit code={circleGridExample} scope={scope} />
+      <LiveEdit code={heartExample} scope={scope} />
```

Comment:
`heartExample` should be at the last since it is newly added.

Reply:
okay